spec/html-proofer/fixtures/links/link_with_me.html

```diff
@@ -0,0 +1,9 @@
+<html>
+<head>
+    <link rel="me" href="https://github.com/gjtorikian/html-proofer"/>
+    <link rel="webmention" href="https://webmention.io/username/webmention" />
+    <link rel="pingback" href="https://webmention.io/username/xmlrpc" />
```

Comment:
Haven't decided yet if this or #529 are preferable, but these should all be faked/broken/garbage links, to prove that they are being skipped.

Reply:
good point, I can fix that if you prefer this solution over #529.